### PR TITLE
Linting and other issues fixed

### PR DESCRIPTION
**Pull request by mscuthbert**
Note that some tests in common.py are still not running -- this is an issue to bring up with the authors (mostly filepath errors)

### CODE REVIEW

File: gender_novels/common.py [hunk 1]
```diff
@@ -152,21 +152,25 @@ def load_file_remotely(file_path, current_file_type):
             # has \r and \n -> replace with only \n
             return text.replace('\r\n', '\n')
 
-def get_text_file_encoding(filepath):
+
+def get_encoding_type(filepath):
```
Comment (srisi) [inline gender_novels/common.py]: What's the argument for changing get_text_file_encoding back to get_encoding_type? I had changed the name to make it obvious that this function only works with text files. I am also ok with get_encoding_type but in that case it should also be changed in corpus_gen.py

File: gender_novels/corpus.py [hunk 2]
```diff
@@ -13,20 +14,38 @@ class Corpus(common.FileLoaderMixin):
...
     """
 
     def __init__(self, corpus_name=None):
         self.corpus_name = corpus_name
         self.novels = []
         if corpus_name is not None:
+            self.relative_corpus_path = Path('corpora', self.corpus_name)
             self.novels = self._load_novels()
 
+    def __len__(self):
```
Comment (srisi) [inline gender_novels/corpus.py]: There's already a __len__function below. Feel free to delete either.
PS: Is there a default order in which the overloaded operators should appear in a class?

File: gender_novels/common.py (hunk 1 above)
Comment (mscuthbert) [reply gender_novels/common.py]: That seems to be a mistaken merge. I didn’t change that. Will revert.

File: gender_novels/corpus.py (hunk 2 above)
Comment (mscuthbert) [reply gender_novels/corpus.py]: Again a merge issue (I added last week when thrrr wasn’t one). When we get lint up, a duplicated method overloaded in the same class is a bug. I have all this set up on eclipse to find it but

File: gender_novels/common.py (hunk 1 above)
Comment (mscuthbert) [reply gender_novels/common.py]: Ah -- I had fixed the doctest for the older function and that overwrote the changes.  Fixing for new name.

**Comment by srisi** [comment]
Thanks!

**Pull request merged**